### PR TITLE
Fixed Documentation regarding installation of kfctl on windows using WSL

### PR DESCRIPTION
**Pull request by sulabh2000**
I have added a note in AWS, AZURE, and IBM kubeflow installation docs regarding the installation of kfctl on windows using WSL as it is available for Linux and macOS users.

### CODE REVIEW

**Comment by PatrickXYS** [comment]
Thanks, @sulabh2000 

I want to confirm whether we succeeded in test windows by using WSL or not, in a word, do we have a manual test for this?

File: content/en/docs/aws/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/aws/deploy/install-kubeflow.md:72]: Nit:

```suggestion
**Note**: kfctl is currently available for Linux and macOS users only. If you use Windows, you can install kfctl on Windows Subsystem for Linux (WSL). Refer to the official [instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10) for setting up WSL.
```

cc @RFMVasconcelos @alfsuse

File: content/en/docs/aws/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/aws/deploy/install-kubeflow.md:74]: Nit:

```suggestion
To deploy Kubeflow on your existing Amazon EKS cluster, you need to provide `AWS_CLUSTER_NAME`, `cluster region` and `worker roles`.
```

File: content/en/docs/azure/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/azure/deploy/install-kubeflow.md:90]: Nit:

```suggestion
**Note**: kfctl is currently available for Linux and macOS users only. If you use Windows, you can install kfctl on Windows Subsystem for Linux (WSL). Refer to the official [instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10) for setting up WSL.
```

cc @RFMVasconcelos @alfsuse

File: content/en/docs/ibm/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/ibm/deploy/install-kubeflow.md:134]: Nit:

```suggestion
**Note**: kfctl is currently available for Linux and macOS users only. If you use Windows, you can install kfctl on Windows Subsystem for Linux (WSL). Refer to the official [instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10) for setting up WSL.
```
cc @alfsuse @RFMVasconcelos

File: content/en/docs/ibm/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/ibm/deploy/install-kubeflow.md:138]: Nit:

```suggestion
Run the following commands to set up and deploy Kubeflow:

1. Download the latest kfctl {{% kf-latest-version %}} release from the
```

**Comment by sulabh2000** [comment]
> Thanks, @sulabh2000
> 
> I want to confirm whether we succeeded in test windows by using WSL or not, in a word, do we have a manual test for this?

yes, I have tested kfctl installation on wsl, and it's working fine. Any user working on windows can install wsl as given in Microsoft documentation and then install kfctl with it's help.

**Comment by sulabh2000** [comment]
@8bitmp3 ok...I will do all the changes suggested by you.

File: content/en/docs/ibm/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/ibm/deploy/install-kubeflow.md:141]: Can you also do some minor linting if you have time? This can help with readability, future reviews, and improved UX for KF users.

---

For example, can you please add blank lines between code blocks and normal text paragraphs below? Also, for better Markdown formatting, add `shell` or `bash` to command-line code blocks. 

Lines 139 and 144:

```diff
- 1. Unpack the tar ball
+ 1. Extract the archived TAR file:
+ 
-       ```
+       ```shell
      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz		      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
+ 
      ```
```

(repeat for line 44, if you can, please)

---

Line 151: please add a blank line as follows:

```diff
### Single user
+       
Run the following commands to set up and deploy Kubeflow for a single user without any authentication.
```

same between the lines 204-205, 291-292, 302-303, 308-309

---

Lines 285-287: Can you please split the complex sentence into simpler ones for readability/better UX?


```diff
    **Note**: If any of the parameters changed after the initial Kubeflow deployment, you
-     will need to manually update these parameters in the secret `appid-application-configuration`
+     will need to manually update these parameters in the secret `appid-application-configuration`.
-     then restart authservice by running the command `kubectl rollout restart sts authservice -n istio-system`.
+     Then, restart authservice by running the command `kubectl rollout restart sts authservice -n istio-system`.
```

---

Line 298: Please change the order of words and fix the grammar:

```diff
## Next steps
+       
- Please follow the steps in [Exposing the Kubeflow dashboard with DNS and TLS termination]
- (../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination) to secure the Kubeflow 
- dashboard with HTTPS, then you will have the required DNS name as Kubeflow FQDN to enable the OIDC flow for 
- AppID:		
+ To secure the Kubeflow dashboard with HTTPS, follow the steps in 
+ [Exposing the Kubeflow dashboard with DNS and TLS termination](../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination).
+ Then, you will have the required DNS name as Kubeflow FQDN to enable the OIDC flow for AppID:

---

Line 308: Please change to capital "R" in "restart" (use sentence case, basically):

```diff
- 3. restart the pod `authservice-0`:		
+ 3. Restart the pod `authservice-0`:		
```

---

Please rephase line 313:

```diff
- Then visit `https://<kubeflow-FQDN>/`, it should redirect you to AppID for authentication.
+ Then, visit `https://<kubeflow-FQDN>/`. The page should redirect you to AppID for authentication.

(or "site" or whatever you think is best.)
```

---

Thank you!

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Hi @sulabh2000 Thanks again! 

Can you also do some minor linting if you have time? This can help with readability, future reviews, and improve the UX for KF users.

---

For example, can you please add blank lines between code blocks and normal text paragraphs below? Also, for better Markdown formatting, add `shell` or `bash` to command-line code blocks. 

Lines 139 and 144:

```diff
- 1. Unpack the tar ball
+ 1. Extract the archived TAR file:
+ 
-       ```
+       ```shell
      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz		      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
+ 
      ```
```

(repeat for line 44, if you can, please)

---

Line 151: please add a blank line as follows:

```diff
### Single user
+       
Run the following commands to set up and deploy Kubeflow for a single user without any authentication.
```

same between the lines 204-205, 291-292, 302-303, 308-309

---

Lines 285-287: Can you please split the complex sentence into simpler ones for readability/better UX?


```diff
    **Note**: If any of the parameters changed after the initial Kubeflow deployment, you
-     will need to manually update these parameters in the secret `appid-application-configuration`
+     will need to manually update these parameters in the secret `appid-application-configuration`.
-     then restart authservice by running the command `kubectl rollout restart sts authservice -n istio-system`.
+     Then, restart authservice by running the command `kubectl rollout restart sts authservice -n istio-system`.
```

---

Line 298: Please change the order of words and fix the grammar:

```diff
## Next steps
+       
- Please follow the steps in [Exposing the Kubeflow dashboard with DNS and TLS termination]
- (../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination) to secure the Kubeflow 
- dashboard with HTTPS, then you will have the required DNS name as Kubeflow FQDN to enable the OIDC flow for 
- AppID:		
+ To secure the Kubeflow dashboard with HTTPS, follow the steps in 
+ [Exposing the Kubeflow dashboard with DNS and TLS termination](../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination).
+ Then, you will have the required DNS name as Kubeflow FQDN to enable the OIDC flow for AppID:

---

Line 308: Please change to capital "R" in "restart" (use sentence case, basically):

```diff
- 3. restart the pod `authservice-0`:		
+ 3. Restart the pod `authservice-0`:		
```

---

Please rephase line 313:

```diff
- Then visit `https://<kubeflow-FQDN>/`, it should redirect you to AppID for authentication.
+ Then, visit `https://<kubeflow-FQDN>/`. The page should redirect you to AppID for authentication.

(or "site" or whatever you think is best.)
```

---

Thank you!

**Comment by sulabh2000** [comment]
@8bitmp3 done with linting..please check!

File: content/en/docs/ibm/deploy/install-kubeflow.md
Comment (8bitmp3) [inline content/en/docs/ibm/deploy/install-kubeflow.md:210]: Sorry, missed this one. This word should be a verb - "to set up" - instead of a noun ("a setup"):

```diff
- 1. Setup environment variables:
+ 1. Set up environment variables:
```

Comment (8bitmp3) [reply content/en/docs/ibm/deploy/install-kubeflow.md:210]: Same for line 232 please. Thank you!

```diff
- 2. Setup configuration files:
+ 2. Set up configuration files:
```

**Review by shawnzhu — APPROVED** [review]
looks good for the kubeflow installation on IBM Cloud. Thank you!

**Comment by PatrickXYS** [comment]
Overally lgtm on AWS side

File: content/en/docs/ibm/deploy/install-kubeflow.md
Comment (sulabh2000) [reply content/en/docs/ibm/deploy/install-kubeflow.md:210]: ok I will correct it.

**Comment by 8bitmp3** [comment]
Thanks you!

This PR covers three groups, each of which have their OWNERS - please chase them to give you the `/approve` label. For future PRs, I'd suggest to submit one PR for each workgroup to make things move faster.

As the bot mentioned earlier, check the OWNERS files under (links above) for get the right `approvers` to OK this work, so this PR can get merged.

`content/en/docs/aws/OWNERS`
`content/en/docs/azure/OWNERS`
`content/en/docs/ibm/deploy/OWNERS`

**Comment by 8bitmp3** [comment]
Thanks @PatrickXYS 

/lgtm on my side

PTAL and `/approve` if all is 👍:
/assign @eedorenko @berndverst @animeshsingh 

Thanks all!

File: content/en/docs/ibm/deploy/install-kubeflow.md
Comment (8bitmp3) [reply content/en/docs/ibm/deploy/install-kubeflow.md]: Thanks ⚡

**Comment by sulabh2000** [comment]
thanks, everyone...waiting for the approval from @berndverst @eedorenko @animeshsingh

**Comment by sulabh2000** [comment]
@animeshsingh Please check it..And approve it if everything is fine.

**Comment by sulabh2000** [comment]
Hi, @animeshsingh waiting for your approval.

**Comment by PatrickXYS** [comment]
@Tomcli @pvaneck @adrian555 Can you take a look at IBM change?

**Comment by Tomcli** [comment]
thanks @sulabh2000, LGTM on my side

**Comment by 8bitmp3** [comment]
Thanks @Tomcli Can you please reply with `/approve` so this can get merged? 👍

**Comment by 8bitmp3** [comment]
@Tomcli looks like @animeshsingh is the sole approver for this IBM workgroup. I'd propose your team adds 1 more approver to OWNERS to speed up future PR reviews 🙏

**Comment by animeshsingh** [comment]
well @8bitmp3   @Tomcli is in the reviewer - so an /lgtm is should be good enough from him? Goal of approver is to explicitly merge if its specific to IBM here..... but this is generic kfctl install instructions, which can be "approved" if he LGTMs.

regardless have added @Tomcli as the approver for IBM docs on a pending PR

**Comment by sulabh2000** [comment]
Thanks, everyone!!

**Pull request merged**
